src/zkp/pisch.rs

```diff
@@ -24,14 +24,20 @@ use utils::CurvePoint;
 
 #[derive(Debug, Serialize, Deserialize)]
 pub(crate) struct PiSchProof {
+    /// Commitment to the secret (`A` in the paper).
     pub(crate) commitment: CurvePoint,
+    /// Fiat-Shamir challenge (`e` in the paper).
     challenge: BigNumber,
+    /// Response binding the commitment randomness used in the commitment (`z`
+    /// in the paper).
     response: BigNumber,
 }
 #[derive(Debug, Serialize, Deserialize)]
 pub(crate) struct PiSchPrecommit {
+    /// Precommitment value (`A` in the paper).
     pub(crate) precommitment: CurvePoint,
-    alpha: BigNumber,
+    /// Randomness mask for commitment (`alpha` in the paper).
+    randomness_for_commitment: BigNumber,
```

Comment:
This is such a useful change. Calling it a mask flags it (to me at least) as something that shouldn't be sent to another party, which makes it way easier to follow the trail that we checked in #368.